components/Visualization/Map/MapContainer/index.js

```diff
@@ -30,11 +32,24 @@ export default class MapContainer extends React.PureComponent {
         );
     }
 
+    resizeMap = memoize((boundingClientRect) => {
```

Comment:
We should use componentWillUpdate or componentDidUpdate and avoid side-effects in render